upgrades/steps_28.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package upgrades
+
+// stateStepsFor272 returns upgrade steps for Juju 2.8.0.
+func stateStepsFor28() []Step {
+	return []Step{
+		&upgradeStep{
+			description: "increment tasks sequence by 1",
+			targets:     []Target{DatabaseMaster},
+			run: func(context Context) error {
+				return context.State().IncrementTasksSequence()
```

Comment:
I feel like we should take the action seq add one and make it the operation id sequence. As in, all current actions should be put under operation entities with the same name. Like action-5 gets a operation-5 with action-5 being the child task.

Reply:
operations and tasks share the same sequence generator
The operation IDs and task IDs must be unique to avoid ambiguity. So we never want operation-5 and action-5. It must always be operation-5 and action-6 (or 7 or 8 or whatever).